docs/zh-cn/1.2.0/user_doc/hardware-environment.md

```diff
@@ -0,0 +1,48 @@
+# 软硬件环境建议配置
+
+DolphinScheduler 作为一款开源分布式工作流任务调度系统，可以很好的部署和运行在 Intel 架构服务器环境及主流虚拟化环境下，并支持主流的Linux操作系统环境。
+
+## 1. Linux 操作系统版本要求
+
+| 操作系统       | 版本         |
+| :----------------------- | :----------: |
+| Red Hat Enterprise Linux | 7.0 及以上   |
+| CentOS                   | 7.0 及以上   |
+| Oracle Enterprise Linux  | 7.0 及以上   |
+| Ubuntu LTS               | 16.04 及以上 |
+
+> **注意：**
+>以上 Linux 操作系统可运行在物理服务器以及 VMware、KVM、XEN 主流虚拟化环境上。
+
```

Comment:
Better to add dependency, such as: database and zookeeper.

Reply:
database and zookeeper requirement has writed in the deployment part